models/cam-se/model_mod.f90

```diff
@@ -1386,8 +1419,8 @@ end subroutine get_se_four_state_values
 !-----------------------------------------------------------------------
 !>
 
-! Returns my_status 0 for success, 16 if unable to find values at lower level
-! and 17 if unable to find values at upper level.
+! Returns my_status 0 for success, 26 if unable to find values at lower level
+! and 27 if unable to find values at upper level.
```

Comment:
What is the significance of 26 vs 16? 27 vs 17? Did these clash with other my_status values?